app/assets/javascripts/app.js

```diff
@@ -19,28 +19,12 @@ planner.run(['$rootScope', function($rootScope){
 
 planner.config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $urlRouterProvider) {
 
-  $urlRouterProvider.otherwise('/');
+  $urlRouterProvider.otherwise('');
 
   $stateProvider
```

Comment:
It does not redirect, and I've just discovered that going to `localhost:3000/` results in a blank screen with our navbar at the top, which is not ideal. However, `$urlRouterProvider.otherwise('/students')` results in severe problems when logged out. Because the login form is on the '/students' page, the browser continually requests advisors.json and students.json and continually gets a stream of 401 Unauthorized from the server. The browser locks up and you have to kill the process.

I have fixed this problem by moving `ng-app="planner"` from app/views/layouts/application.html.erb to app/views/angular_app/index.html.erb. This required moving around some container classes to make it look OK. `$urlRouterProvider.otherwise` is set to `/students` again.